src/cmd.rs

```diff
@@ -275,6 +279,13 @@ impl Value {
         panic!("expected {:?} to be a `uint`", self);
     }
 
+    pub fn color(&self) -> Rgba8 {
+        if let Value::Rgba8(rgba8) = self {
+            return *rgba8 as Rgba8;
```

Comment:
Simply `return rgba8` should be sufficient here, as we're not casting.

Reply:
Done.